pennylane/labs/resource_estimation/ops/op_math/symbolic.py

```diff
@@ -0,0 +1,217 @@
+            pass
+
+        gate_types = {}
+
+        if num_ctrl_values == 0:
+            decomp = base_class.resources(**base_params, **kwargs)
+            for gate, count in decomp.items():
+                rep = cls.resource_rep(gate.op_type, gate.params, num_ctrl_wires, 0, num_work_wires)
+                gate_types[rep] = count
+
+            return gate_types
+
+        no_control = cls.resource_rep(base_class, base_params, num_ctrl_wires, 0, num_work_wires)
+        x = re.ResourceX.resource_rep()
+        gate_types[no_control] = 1
+        gate_types[x] = 2 * num_ctrl_values
+
+        return gate_types
+
+    def resource_params(self) -> dict:
+        return {
+            "base_class": type(self.base),
+            "base_params": self.base.resource_params(),
+            "num_ctrl_wires": len(self.control_wires),
+            "num_ctrl_values": len([val for val in self.control_values if not val]),
+            "num_work_wires": len(self.work_wires),
+        }
+
+    @classmethod
+    def resource_rep(
+        cls, base_class, base_params, num_ctrl_wires, num_ctrl_values, num_work_wires
+    ) -> re.CompressedResourceOp:
+        return re.CompressedResourceOp(
+            cls,
+            {
+                "base_class": base_class,
+                "base_params": base_params,
+                "num_ctrl_wires": num_ctrl_wires,
+                "num_ctrl_values": num_ctrl_values,
+                "num_work_wires": num_work_wires,
+            },
+        )
+
+    @classmethod
+    def controlled_resource_decomp(
+        cls,
+        outer_num_ctrl_wires,
+        outer_num_ctrl_values,
+        outer_num_work_wires,
+        base_class,
+        base_params,
+        num_ctrl_wires,
+        num_ctrl_values,
+        num_work_wires,
+    ) -> Dict[re.CompressedResourceOp, int]:
+        return {
+            cls.resource_rep(
+                base_class,
+                base_params,
+                outer_num_ctrl_wires + num_ctrl_wires,
+                outer_num_ctrl_values + num_ctrl_values,
+                outer_num_work_wires + num_work_wires,
+            ): 1
+        }
+
+    @staticmethod
+    def tracking_name(base_class, base_params, num_ctrl_wires, num_ctrl_values, num_work_wires):
+        base_name = base_class.tracking_name(**base_params)
+        return f"C({base_name},{num_ctrl_wires},{num_ctrl_values},{num_work_wires})"
+
+
+class ResourcePow(PowOperation, re.ResourceOperator):
+    """Resource class for Pow"""
+
+    @staticmethod
+    def _resource_decomp(
+        base_class, z, base_params, **kwargs
+    ) -> Dict[re.CompressedResourceOp, int]:
+        try:
+            return base_class.pow_resource_decomp(z, **base_params)
+        except re.ResourcesNotDefined:
+            pass
+
+        try:
+            return _scale_dict(base_class.resources(**base_params, **kwargs), z)
+        except re.ResourcesNotDefined:
+            pass
+
+        return {base_class.resource_rep(): z}
```

Comment:
I don't think we need this second try block. I think we should just return z times the resource rep of the base and then let `get_resources` recursively expand the ResourcePow if needed. 
```suggestion
        return {base_class.resource_rep(**base_params, **kwargs): z}
```

Reply:
I agree. I actually thought I had removed that already, but I guess not?